src/pages/gift/accounts/LedgerWallet/LedgerInstructions.js

```diff
@@ -1,77 +1,64 @@
   const { giftTheme } = useSubstrate();
   return (
     <>
-      <Card.Body>
+      <Card.Body className="d-flex flex-column">
         <CardHeader
           title={title}
           backClickHandler={prevStepHandler}
           cardText="Follow below instructions to import your Ledger account into Polkadot extension."
         />
-        <Row className="py-5 justify-content-center">
-          <Col className="d-flex flex-column justify-content-center align-items-center">
-            <div>
-              <ul>
-                <li className="mb-5">
-                  <div className="mb-3">
-                    <span className="step-text">
-                      {`Install ${giftTheme?.network} App and create a ${giftTheme?.network} account on your
-                      Ledger device. `}
-                    </span>
-                    <a
-                      href="https://support.ledger.com/hc/en-us/articles/360016289919-Polkadot-DOT-"
-                      rel="noreferrer"
-                      target="_blank">
-                      learn more
-                    </a>
-                  </div>
-                  <div className="d-flex flex-column">
-                    <Image
-                      src={ledgerInstall}
-                      className="rounded align-self-end image-box"
-                    />
-                  </div>
-                </li>
-                <li className="step-item">
-                  <div className="mb-3">
-                    <span className="step-text">
-                      Add your Ledger account to Polkadot extension.&nbsp;
-                    </span>
-                    <a
-                      href={`${extensionLink}`}
-                      rel="noreferrer"
-                      target="_blank">
-                      learn more
-                    </a>
-                  </div>
-                  <div className="d-flex flex-column">
-                    <Image
-                      src={extensionAdd}
-                      className="rounded align-self-end image-box mb-2"
-                    />
-                    <Image
-                      src={extensionLedger}
-                      className="rounded align-self-start image-box"
-                    />
-                  </div>
-                </li>
-              </ul>
-            </div>
+        <Row className="justify-content-center">
+          <Col sm="11">
+            <CardLink
+              hRef="https://support.ledger.com/hc/en-us/articles/360016289919-Polkadot-DOT"
+              title="Step 1"
+              subtitle={`Install the ${giftTheme?.network} on your Ledger device`}
+            />
+          </Col>
+        </Row>
+        <Row className="justify-content-center py-3">
+          <Col sm="11">
+            <CardLink
+              hRef={extensionLink}
+              title="Step 2"
+              subtitle="Add your Ledger to the Polkadot Extension"
```

Comment:
They need to add their "Ledger account". must be:
"Add your Ledger account to the Polkadot Extension"